.gitignore

```diff
@@ -34,3 +34,6 @@ docs/auto_examples
 docs/examples/graph*.*
 docs/examples/rf_iris.onnx
 docs/examples/logreg_iris.onnx
+tests/TESTDUMP
+tests/tests/
```

Comment:
given the tests folder is already used to be a unit tests folder, please change the name of the dump data folder.